index.html

```diff
@@ -9,27 +9,66 @@
 </head>
 <body>
 
+    <div class="nav-bar">
+        <div class="nav-bar-title">Awesome Books</div>
+        <nav>
+        <ul class="list-add-contact">
+            <li class="nav-list">List</li>
+            <hr>
+            <li class="nav-add">Add New</li>
+            <hr>
+            <li class="nav-contact">Contact</li>
+        </ul>
+        </nav>
+    </div>
+    <!-- <button class="bu" type="button" onclick="displayDate()">Date & Time</button> -->
+    <p id="showDate"> Wenseday May 3, 2023 </p>
+    <!-- first tab ///////////////////////////////////////// -->
+    
```

Comment:
- [ ] it would be better to remove the booklist from add book section in order to match the wireframe 
- Yours 
<img width="1431" alt="Screenshot 2023-05-03 at 19 35 47" src="https://user-images.githubusercontent.com/61790587/236011979-d58bcf47-596b-46ee-a6d3-fffa1c41ff1d.png">
- Wireframe
<img width="962" alt="Screenshot 2023-05-03 at 19 05 44" src="https://user-images.githubusercontent.com/61790587/236012032-0269ec70-4657-4bbf-93fd-25d5ce4e2589.png">